storybookjs/storybook
# Source-loader: Fix Typescript support

Issue: #8198

## What I did

A bunch of bugfixes for source-loader typescript support. AFAIK source-loader now works out of the box, at least with the CRA typescript preset.

## How to test

- See updated tests/snapshots
- Run `cra-ts-kitchen-sink`	& observe source in `Docs` tab
